lib/services/redmine/redmine_issue_resource.rb

```diff
@@ -23,10 +23,10 @@ def update(payload_fragment: nil, parent_id: nil, version_id: nil)
     params[:issue].delete :priority_id # Don't override priority on upload
     prepare_request
     response = http_put redmine_issues_path(issue_id), params.to_json
-    process_response response, 200 do
+    process_response response, 200, 204 do
```

Comment:
Redmine sometimes returns 204s? That's...an interesting choice on their part...

Reply:
yeah, I can't find documentation about it, but it looks like it happens for updates. 🤷